powersimdata/network/hifld/constants/plants.py

```diff
@@ -0,0 +1,105 @@
+_exports = [
+    "all_resources",
+    "carbon_per_mmbtu",
+    "carbon_per_mwh",
+    "carbon_resources",
+    "clean_resources",
+    "label2type",
+    "nox_per_mwh",
+    "renewable_resources",
+    "so2_per_mwh",
+    "type2color",
+    "type2hatchcolor",
+    "type2label",
+]
+
+type2color = {
+    "wind": "xkcd:green",
+    "solar": "xkcd:amber",
+    "hydro": "xkcd:light blue",
+    "ng": "xkcd:orchid",
+    "nuclear": "xkcd:silver",
+    "coal": "xkcd:light brown",
+    "geothermal": "xkcd:hot pink",
+    "dfo": "xkcd:royal blue",
+    "biomass": "xkcd:dark green",
+    "other": "xkcd:melon",
+    "storage": "xkcd:orange",
+    "wind_offshore": "xkcd:teal",
+    "solar_curtailment": "xkcd:amber",
+    "wind_curtailment": "xkcd:green",
+    "wind_offshore_curtailment": "xkcd:teal",
+}
+
+type2label = {
+    "nuclear": "Nuclear",
+    "geothermal": "Geo-thermal",
+    "coal": "Coal",
+    "dfo": "DFO",
+    "hydro": "Hydro",
+    "ng": "Natural Gas",
+    "solar": "Solar",
+    "wind": "Wind",
+    "wind_offshore": "Wind Offshore",
+    "biomass": "Biomass",
+    "other": "Other",
+    "storage": "Storage",
+    "solar_curtailment": "Solar Curtailment",
+    "wind_curtailment": "Wind Curtailment",
+    "wind_offshore_curtailment": "Offshore Wind Curtailment",
+}
```

Comment:
I know that PostREISE plotting functions rely on this dictionary and it is convenient to group generator type and curtailment but it is ugly. Curtailment color label/color should be moved in a dedicated dictionary. This completely out of the scope of this PR.